packagingrunner/docker-entrypoint.sh

```diff
@@ -8,7 +8,12 @@ operation="${1:-complete}"
 
 case "$operation" in
 build)
-  bundle exec rake build:all
+  shift
+  if [ "$#" -gt 0]; then
+    bundle exec rake build "$@"  # should be able to pass st2 or st2mistral as packages_list but can't figure this rake part.
```

Comment:
it's not required, this doesn't make sense unless you want execute custom tasks with rake. as above^^
`bundle exec rake task1 task2` - that what it means.
However `docker-compose run -e ST2_PACKGES=st2mistral el6 build` is already a sufficient and required condition to achieve just **st2mistral** building